### DISCUSSION TITLE
🆕 Billing for Codespaces will begin September 13, 2021

### DISCUSSION THREAD

**Discussion by bailey** [community/community#5588] [Enterprise]
Codespaces is currently free to use for all organizations on a GitHub Team or GitHub Enterprise Cloud plan. On **Monday, September 13**, billing will begin and your organization’s or enterprise’s Codespaces spending limit will be set to $0. This will prevent new codespaces from being created or existing codespaces from being opened. 

To allow your users to continue using Codespaces in your organization or enterprise, you or an enterprise administrator will need to [**set the limit**](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/managing-spending-limits-for-codespaces) **to a value higher than $0 prior** to September 13, 2021.

**Comment by bonellia** [comment]
I am using Codespaces (I had opted-in early when it was beta) and I am on "Github Student Pack" version of pro. I am not sure if this is included in GithHub Team or anything. Will I be able to use codespaces on regular GitHub pro plan? Thanks in advance.

**Reply by bailey** [reply]
GitHub Codespaces is not generally available to individuals at this point. To use Codespaces, you'll need to be part of an organization with a GitHub Team or GitHub Enterprise Cloud plan that has set a spending limit for Codespaces.

**Comment by treeder** [comment]
Did this go live?  Why are we still limited to 10 containers?

**Reply by tanmayeekamath** [reply]
Hey @treeder, yes. What's an expected limit based on your use case?

**Reply by treeder** [reply]
I think this is no longer limited, it's all good now.
